WMF Framework/WMFCaptchaViewController.swift

```diff
@@ -4,6 +4,23 @@ import UIKit
 // Presently it is assumed this view controller will be used only as a
 // child view controller of another view controller.
 
+extension UIView {
+    fileprivate func wmf_subviews(hide: Bool) {
```

Comment:
hmm unsure what the set/get would add... presently this recursively hides subviews...